utils.go

```diff
@@ -874,26 +874,63 @@ func getInode(file string) (uint64, error) {
 }
 
 func checkIDMapMountSupport(ctx *cli.Context) (bool, bool, error) {
+	// The sysbox lib dir may have restrictive permissions; loosen those up
+	// temporarily while we perform the shiftfs check.
+	fi, err := os.Stat(sysboxLibDir)
+	if err != nil {
+		return false, false, err
+	}
+	origPerm := fi.Mode()
+
+	if err := os.Chmod(sysboxLibDir, 0755); err != nil {
+		return false, false, fmt.Errorf("failed to chmod %s to 0755: %s", sysboxLibDir, err)
+	}
+
```

Comment:
I think it would be wise to add a `defer` instruction in line 891 and 896 to revert the `chown`, otherwise, in the next boot up attempt, we might end up leaving `/var/lib/sysbox` as `755` for good.

Reply:
Actually, just noticed that we're eliminating `sysboxLibDir` during the boot up sequence, so the issue I highlighted above may not be applicable.

Reply:
Thanks; I still added the defer() because it's cleaner.